include/smack/smack.h

```diff
@@ -58,7 +66,151 @@ int __SMACK_nondet() {
 void __SMACK_decls() {
 #define D(d) __SMACK_top_decl(d)
 
+// SMACK type definitions
+#ifdef BITVECTOR
```

Comment:
**Note** the declarations of `$NULL` and `$UNIT` should also be generated in `Smackrep::getPrelude()`.